roles/edpm_update/tasks/main.yml

```diff
@@ -19,12 +19,21 @@
 
 - name: Apply kernel patch via kpatch
   ansible.builtin.include_tasks: kpatch.yml
-  when: edpm_update_enable_kpatch
+  when:
+  - edpm_update_enable_kpatch
+  - not ansible_local.bootc
 
 - name: Update packages
   ansible.builtin.include_tasks: packages.yml
-  when: edpm_update_enable_packages_update
+  when:
+  - edpm_update_enable_packages_update
+  - not ansible_local.bootc
 
 - name: Update containers
```

Comment:
We plan to split away the "Update packages" from "Update containers" into two distinct steps in the user minor update workflow, as users want more control and the ability to only replace containers without triggering OS-level updates:

https://issues.redhat.com/browse/RHOSSTRAT-23 
https://issues.redhat.com/browse/OSPRH-13841 

Based on your experience with bootc updates so far, do you think running the "Update containers" step in a different Deployment CR from "Update OS (bootc)" step would cause some kind of problem? It looks to me like we should be able to make these independent regardless if we use package mode or image mode.

Reply:
Yeah, I think it can be independent with both of them without issue. The thing to remember with image mode is that it's not _actually_ updating any packages, but rather downloading them and adding them to a different ostree layer. So nothing will be updated until the node has been rebooted. That's the major different between the two, so in that regard, it doesn't really matter too much when it happens, as long as it does happen at some point.